cli/magic-config.ts

```diff
@@ -322,15 +338,22 @@ async function processCreateOptions(options: any): Promise<void> {
   }
   const modelsPrompts = [
     {
-      type: "select",
+      type: "select", 
       name: "defaultEmbedding",
-      message: "Which is the default embedding model",
-      choices: embeddingModels.map((m) => ({ name: m.name, value: m })),
-      initial: options.defaultEmbedding || undefined,
-      skip(): boolean {
-        return !(this as any).state.answers.enableRag;
+      message: "Select a default embedding model",
+      choices: embeddingModels.map(m => ({name: m.name, value: m})),
+      initial: options.defaultEmbedding,
+      validate(value: string) {
+        if ((this as any).state.answers.enableRag) {
+          return value ? true : 'Select a default embedding model'; 
+        }
+      
+        return true;
       },
-    },
+      skip() {
+        return !answers.enableRag; 
```

Comment:
this is wrong as if you select only kendra as your RAG you dont need an embedding model.

Reply:
All good, fair point. Made a change so it only shows when it's aurora and/or opensearch.